new/LIT7302JacSarAnnunciation.xml

```diff
@@ -0,0 +1,85 @@
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7302JacSarAnnunciation" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">Homily on the Annunciation</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="MKr"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>In Ethiopian homiliaries the translation of Jacob of Sarug's Homily on the Annunciation was divided into three parts to cover three liturgical occasions i. e. the first three Sundays of Advent. </p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Homily"/>
+                    <term key="Translation"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="MKr" when="2025-02-26">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                
+             
+                    <listBibl type="editions">
+                        <bibl><ptr target="bm:Tedros2019Annunciation"/></bibl>
+                    </listBibl>    
+                
+                
+                <listRelation>
+                    <relation name="saws:isAttributedToAuthor" active="LIT7302JacSarAnnunciation" passive="PRS5655Jacobof"/>
+                    <relation name="ecrm:P129_is_about" active="LIT7302JacSarAnnunciation" passive="PRS6819Mary"/>
```

Comment:
I would also add the parts as a relation and/or also to the abstract, for more visibility from the first page of the record